docs/docs/launch-manual/sections/using-pulsar/sections/atom-packages.md

```diff
@@ -0,0 +1,133 @@
+### Pulsar Packages
```

Comment:
I'd rename this file. It's going to be confusing that it's named atom-packages even if it's just as a include.

Reply:
Its just a draft to get it shown as being worked on, not even nearly finished yet.